src/components/views/voip/PictureInPictureDragger.tsx

```diff
@@ -0,0 +1,208 @@
+/*
+Copyright 2021 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import React, { createRef } from 'react';
+import UIStore from '../../../stores/UIStore';
+import { IApp } from '../../../stores/WidgetStore';
+import { lerp } from '../../../utils/AnimationUtils';
+import { MarkedExecution } from '../../../utils/MarkedExecution';
+import { replaceableComponent } from '../../../utils/replaceableComponent';
+
+const PIP_VIEW_WIDTH = 336;
+const PIP_VIEW_HEIGHT = 232;
+
+const MOVING_AMT = 0.2;
+const SNAPPING_AMT = 0.05;
+
+const PADDING = {
+  top: 58,
+  bottom: 58,
+  left: 76,
+  right: 8,
+};
+
+interface IProps {
+  className?: string;
+    children: (startMovingEventHandler: (event: React.MouseEvent<Element, MouseEvent>) => void) => React.ReactNode;
+  draggable: boolean;
+  app?: IApp;
```

Comment:
Seems unused?

Reply:
Leftover from the widget part, can be potentially removed.

Reply:
If it's unused, we should remove it, I think